### ISSUE TITLE
A11y_Visual Studio Code Client_Welcome Screen_Label in Name:The incorrect label name is defined for the checkbox as "when checked this page will be shown on startup"

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#160397]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG2.5.3; #Label in Name; #A11yWCAG2.1;

### Environment and OS details:
Visual Studio Code
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:
1)Open Visual studio code editor
2)Using "Alt+H" key navigate to the "Help" menu bar option .
3)Using "Down" arrow key navigate to the "Welcome" item.
4) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
5)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the screen reader is announcing the correct label name of the "show welcome page on startup".

### Actual Result:
The incorrect label name is defined for the checkbox as "when checked this page will be shown on startup".

### Expected Result:
The correct label name should be defined for the checkbox as "show welcome page on startup". Visual name and programmatic name should be same like: Show welcome page on startup.
![A11y_Visual Studio Code Client_Welcome Screen_Label in Name](https://user-images.githubusercontent.com/98530069/189108351-c77d8f6f-4b7e-4f98-ba21-418364a6e835.png)

**Comment by isidorn** [comment]
Disagree. The description is the correct one. They are synonims and it is a product decision to choose which one to show on hover and to screen readers. 
Closing as invalid
